sqlglot/dialects/bigquery.py

```diff
@@ -1042,3 +1035,13 @@ def version_sql(self, expression: exp.Version) -> str:
             if expression.name == "TIMESTAMP":
                 expression.set("this", "SYSTEM_TIME")
             return super().version_sql(expression)
+
+        @generator.unsupported_args("group", "parameters")
```

Comment:
I added this validation for unsupported args, this wasn't strictly needed for this PR but I thought it was still a positive change.